drivers/mysql.go

```diff
@@ -552,6 +519,7 @@ func (db *MySQL) ExecutePendingChanges(changes []models.DbDmlChange) (err error)
 	if err != nil {
 		return err
 	}
+	defer trx.Rollback()
```

Comment:
What is this error useful though? What you gonna do with failed rollback? Except logging? 

The rollback fails if the connection dies ⇾ the db engine will roll back internally, or when the transaction times out.
Also in this setup the Rollback will mostly return `sql.ErrTxDone`, because the transaction was committed.

Reply:
The output of failed rollback is not changes were applied, which is what you wanna achieve with rollback, no? :D

I added the Rollback because it closes the uncommitted transaction earlier. The old code kept the transaction open until it times out.

Reply:
I already faced an SQL transaction that failed to rollback.

And we were happy to see them popping out in errors, so we were able to look further.

Reply:
Ok, given the fact that lazysql may support various database engines with various isolation levels, the error message may come handy sometimes.